### ISSUE TITLE
Windows 10 skype app accessibility issues

### ISSUE THREAD

**Issue by hozosch** [nvaccess/nvda#8534]
I'm sorry if this issue has already been raised, I didn't want to look through all these
### Steps to reproduce:
1. Open the windows 10 skype app
2. Press alt 1.
### Actual behavior
NVDA responds very slowly in the list box.
### Expected behavior:
NVDA responds imediatelly after a key press.
#### NVDA Installed/portable/running from source:
installed
#### NVDA version:
2018.2.1
#### Windows version:
10.0.17134 workstation
#### Name and version of other software in use when reproducing the issue:
Windows 10 built in skype

**Comment by josephsl** [comment]
Hi, Alt+1 moves to conversations, and Alt+2 is contacts. I cannot reproduce this problem on my computer, but I’ll keep an eye on it. Thanks.

**Comment by hozosch** [comment]
O, thanks for the info. I can't even access the actual contacts list! I create an issue for that if it hasn't been created allready.

**Comment by josephsl** [comment]
Hi, I think we can keep it here (just changing the title, that’s all). Regarding contacts list, you need to tab around until you land on the list of contacts. Thanks.

**Comment by hozosch** [comment]
O yea. It's just announcing the tap control again before the list is announced, this confused me.

**Comment by Adriani90** [comment]
@hozosch I am testing with NVDA 2019.1.1 and Skype UWP (Windows store app built in Windows) 8.42. I cannot reproduce this issue. are you still having it?

**Comment by Adriani90** [comment]
We do not have any answer from the initial author, I am closing this as works for me. if you still have this problem, please comment on it and we can reopen the discussion. Thanks.
